assets/src/components/mapPage/routePropertiesCard.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useState } from "react"
+import { useRoute } from "../../contexts/routesContext"
+import { CircleCheckIcon } from "../../helpers/icon"
+import {
+  RoutePattern,
+  RoutePatternId,
+  DirectionName,
+  DirectionId,
+  RouteId,
+  Route,
+  ByRoutePatternId,
+} from "../../schedule"
+import { Card } from "../card"
+import CloseButton from "../closeButton"
+import { RoutePill } from "../routePill"
+
+const sortRoutePatterns = (routePatterns: RoutePattern[]): RoutePattern[] =>
+  routePatterns.sort((rp1, rp2) => rp1.sortOrder - rp2.sortOrder)
+
+const displayName = (routePattern: RoutePattern) => {
+  //const splitName = routePattern.name.split("- ", 2)
+  return routePattern.name
+}
+
+const DetailSection = ({
+  title,
+  isOpen,
+  toggleOpen,
+  className,
+  children,
+}: {
+  title: string
+  isOpen: boolean
+  toggleOpen: () => void
+  className: string
+  children: JSX.Element
+}) => {
+  return (
+    <details
+      className={`m-route-properties-card__details-section ${className}`}
+      {...(isOpen ? { open: true } : {})}
+    >
+      <summary
+        onClick={(event) => {
+          // preventDefault so that open/closed state is managed entirely by react
+          // see https://github.com/facebook/react/issues/15486
+          event.preventDefault()
+          toggleOpen()
+        }}
+      >
+        {`${isOpen ? "Hide" : "Show"} ${title}`}
```

Comment:
I dunno if I'm changing the logic here with this suggestion

```suggestion
        {isOpen ? "Hide" : "Show"} {title}
```